tests/system/trace/basic_trace/basic_trace_system_test.py

```diff
@@ -65,8 +63,15 @@ def test_tracer(self):
         self.assertEqual(trace_json.get('traceId'), trace_id)
         self.assertEqual(len(spans), 2)
 
+        self.assertSetEqual(
+            {ss['displayName']['value'] for ss in spans},
+            {'child_span', 'root_span'})
+
         for span in spans:
-            if span.get('displayName').get('value') == 'root_span':
-                self.assertEqual(str(span.get('parentSpanId')), span_id)
+            if span['displayName']['value'] == 'root_span':
+                self.assertEqual(span['parentSpanId'], span_id)
+                self.assertEqual(span['childSpanCount'], 1)
             else:
-                self.assertEqual(span.get('parentSpanId'), parent_span_id)
+                self.assertEqual(span['displayName']['value'], 'child_span')
```

Comment:
This test could have caught the bug: both exported spans were the same (root) span here.